app/policies/application_policy.rb

```diff
@@ -79,6 +79,14 @@ def superadmin_or_volunteer_related?
     superadmin? || volunteer_related?
   end
 
+  def admin_or_department_manager_or_assignment_related?
+    superadmin_or_department_manager? || volunteer_related?
+  end
+
+  def admin_or_department_manager_or_volunteer_related?
```

Comment:
The naming of this function and of the above one are a bit confusing. There is `volunteer_related` in the title here but `volunteer_related?` is called in the above function. Also (I don't know if RuboCop is a reason for that) but since we only have superadmins and not admins, maybe the naming should be `supeadmin_or...`